streaming/src/main/scala/org/apache/spark/streaming/dstream/FileInputDStream.scala

```diff
@@ -126,6 +126,8 @@ class FileInputDStream[K, V, F <: NewInputFormat[K,V]](
   @transient private var path_ : Path = null
   @transient private var fs_ : FileSystem = null
 
+  protected override val customScopeName: Option[String] = Some(s"file stream [$id]")
```

Comment:
Technically, this should/could be either `file stream` and `text file stream` depending on whether `ssc.fileStream` or `ssc.textFileStream` created it.  But I am okay as is.